subworkflows/local/generate_families.nf

```diff
@@ -0,0 +1,91 @@
+/*
+    FAMILY MODEL GENERATION
+*/
+
+include { ALIGN_SEQUENCES  } from '../../subworkflows/local/align_sequences'
+include { CLIPKIT          } from '../../modules/nf-core/clipkit/main'
+include { CLIP_ENDS        } from '../../modules/local/clip_ends.nf'
+include { HMMER_HMMBUILD   } from '../../modules/nf-core/hmmer/hmmbuild/main'
+include { HMMER_HMMSEARCH  } from '../../modules/nf-core/hmmer/hmmsearch/main'
+include { FILTER_RECRUITED } from '../../modules/local/filter_recruited.nf'
+include { HMMER_HMMALIGN   } from '../../modules/nf-core/hmmer/hmmalign/main'
+
+workflow GENERATE_FAMILIES {
+    take:
+    sequences // tuple val(meta), path(fasta)
+    fasta_chunks
+
+    main:
+    ch_versions = Channel.empty()
+    ch_msa      = Channel.empty()
+    ch_fasta    = Channel.empty()
+    ch_hmm      = Channel.empty()
+
+    fasta_chunks
+        .transpose()
+        .map { meta, file_path ->
+            [ [id: meta.id, chunk: file(file_path).baseName], file_path ]
+        }
+        .set { ch_fasta }
```

Comment:
In most cases you are using `ch_<name> = ` for assigning channels, use that notation rather than `.set{}` for consistency